solutions/search/serverless-elasticsearch-get-started.md

```diff
@@ -55,14 +55,24 @@ Use your {{ecloud}} account to create a fully-managed {{es}} project:
 6. Once the project is ready, select **Continue**.
 
 ::::{tip}
+
+**The General Purpose (GP) profile is recommended for most search use cases such as:**
+
+- full text search
```

Comment:
Hyphen is used in https://www.elastic.co/docs/solutions/search/full-text so ideally both should be aligned one way or the other

```suggestion
- full-text search
```